tests/plot.test.tsx

```diff
@@ -0,0 +1,153 @@
+import { test, expect } from '@playwright/experimental-ct-react';
+
+import {
+  Annotations,
+  Axis,
+  BarSeries,
+  FunctionSeries,
+  Heading,
+  Legend,
+  LineSeries,
+  Plot,
+  RangeSeries,
+  ScatterSeries,
+} from '../src';
+import { Line } from '../src/components/Annotations/Line';
+
+import { InfraredPlotTest } from './utils';
+
+const DEFAULT_PLOT_CONFIG = {
+  width: 900,
+  height: 540,
+};
+const data = [
+  {
+    x: 0,
+    y: 10,
+  },
+  {
+    x: 1,
+    y: 12,
+  },
+  {
+    x: 2,
+    y: 14,
+  },
+  {
+    x: 3,
+    y: 16,
+  },
+  {
+    x: 4,
+    y: 18,
+  },
+];
+
+const rangeData = [
+  {
+    x: 1,
+    y1: -3,
+    y2: 4,
+  },
+  {
+    x: 2,
+    y1: -2,
+    y2: 5,
+  },
+  {
+    x: 3,
+    y1: 2,
+    y2: 10,
+  },
+  {
+    x: 4,
+    y1: 7,
+    y2: 16,
+  },
+  {
+    x: 5,
+    y1: 12,
+    y2: 22,
+  },
+];
+
+test.describe('Plot tests', () => {
+  test('empty plot', async ({ mount }) => {
+    const plot = await mount(
+      <Plot {...DEFAULT_PLOT_CONFIG}>
+        <Axis position="bottom" />
+        <Axis position="left" />
+      </Plot>,
+    );
+    await expect(plot).toBeEmpty();
+    await expect(plot.locator('_react=Axis')).toHaveCount(0);
+  });
+  test('default Plot children with series', async ({ mount }) => {
+    const plot = await mount(<InfraredPlotTest />);
+    const series = plot.locator('_react=ScatterSeries');
+    const xAxis = plot.locator('_react=Axis[position="bottom"]');
+    const yAxis = plot.locator('_react=Axis[position="left"]');
+    await expect(series).toBeEnabled();
```

Comment:
Did you mean to use [`toBeVisible()`](https://playwright.dev/docs/test-assertions#locator-assertions-to-be-visible) ?
I think "enabled" only makes sense on elements that can be disabled, like buttons or inputs.

Reply:
> Did you mean to use [`toBeVisible()`](https://playwright.dev/docs/test-assertions#locator-assertions-to-be-visible) ?

yeah thank you